modules/ui/src/main/scala/lucuma/ui/forms/FormInputEV.scala

```diff
@@ -191,8 +193,11 @@ object FormInputEV {
         }
       )
 
-    val onKeyDown: ReactKeyboardEventFromInput => Callback = e =>
-      $.setStateL(State.lastKeyCode)(e.keyCode) *> clearStateCursor
+    def onKeyDown(props: Props[EV, A], state: State): ReactKeyboardEventFromInput => Callback = e =>
+      if (e.keyCode === 13)
+        submit(props, state)
+      else
+        $.setStateL(State.lastKeyCode)(e.keyCode) *> clearStateCursor
```

Comment:
Good catch. I guess we don't want to do these on `Enter` because Enter is pretty much a no-op as far as the rest of the functionality is concerned.